.gitignore

```diff
@@ -85,3 +85,6 @@ corehq/apps/hqwebapp/static/hqwebapp/js/lib/modernizr.js
 # direnv
 .envrc
 .direnv
+
+# webpack build related
+/webpack/*.json
```

Comment:
I like the idea of moving the webpack build artifacts to _build, however I disagree with moving the directory to a different location. We currently have a non standard way of building requirejs. It is triggered by a django management command not nom or yarn. That is likely the reason why the config is in hqwebapp static, as it makes its path more easily accessible by said management command. These configs have no need for being in a public staticfiles folder.

Putting the webpack config in the same location as requirejs feels unintuitive for folks not immersed in HQ's structure and is definitely not standard according to the majority of documentation and literature out there, which has the webpack config always close to the root of the repository. Additionally, since yarn (and eventually npm) handles the build process not a management command, the path to reference the webpack config will then be unnecessarily long.